components/blocks/Features.tsx

```diff
@@ -1,56 +1,65 @@
-import { Actions } from './Actions';
-import { Container } from './Container';
-import { Prism } from '../styles/Prism';
-import { tinaField } from 'tinacms/dist/react';
-import DocsRichText from '../styles/DocsRichText';
-import styled from 'styled-components';
-import playImage from '../../public/img/playButton.png';
+import { Actions } from './Actions'
+import { Container } from './Container'
+import { Prism } from '../styles/Prism'
+import { tinaField } from 'tinacms/dist/react'
+import DocsRichText from '../styles/DocsRichText'
+import styled from 'styled-components'
+import playImage from '../../public/img/playButton.png'
```

Comment:
Not a fan of removing all the ; semicolons... I guess thats a formatting config option.

Is this consistent with the rest of the project?
I think we should make it consistent with the SSW.Website project (new pbi)